ReadAlongs/Studio
# Convert the documentation from sphinx .rst to mkdocs .md

### PR Goal? 

Convert our documentation from Sphinx to mkdocs

A few updates to the docs were made, but mostly it's just conversion.

### Fixes? 

Fixes #213

### Feedback sought? 

1) Check that the general results are correct (see temporary deployment at https://joanise.github.io/ReadAlong-Studio/dev/)
2) take a look at the code
3) Next, I want to discuss how to change the live deployment without loosing all our previously published versions at https://readalong-studio.readthedocs.io/ where we have five older versions going back to 2021.

### Priority? 

normal

### Tests added? 

Tested locally, and on my fork.

### How to test? 

Locally, check out the PR branch, and run:
```
pip install -r docs/requirements.txt
mike serve
```
then navigate to http://localhost:8000

Or, I'm repeating myself, check my temporary deployment at  https://joanise.github.io/ReadAlong-Studio/dev/

### Confidence? 

Good for the rendering of the current version of the docs.

N/A for how to deploy, since I did not solve that yet.

This PR is ready to review in terms of looking at the current results, but it is not ready to merge since it doesn't deploy.

### Version change? 

no
